minitests/litex/uart_ddr/arty/src.vivado/Makefile

```diff
@@ -0,0 +1,33 @@
+PART = xc7a35tcsg324-1
+BIT2FASM_ARGS = --part "$(XRAY_DIR)/database/artix7/$(PART)" --verbose
+SOURCES = ../verilog/mem.init ../verilog/mem_1.init ../verilog/top.v
+
+all: top.fasm top.bits segprint.log
+
+clean:
+	@rm -f *.bit
+	@rm -f *.bin
+	@rm -f *.bits
+	@rm -f *.fasm
+	@rm -f *.log
+	@rm -rf build
+
+.PHONY: all clean
+
+top.bit: $(VIVADO) $(SOURCES) top.xdc top.tcl
```

Comment:
dependency of this target is `top.tcl` and in the target's command you use `../top.tcl`

Reply:
I've checked, the command first gets to the build directory, therefore this should be fine